Gems/ROS2PoseControl/Code/Source/Clients/ROS2PoseControl.cpp

```diff
@@ -90,32 +90,44 @@ namespace ROS2PoseControl
         }
     }
 
-    AZ::Outcome<AZ::Transform, const char*> ROS2PoseControl::GetCurrentTransformViaTF2()
+    AZ::Outcome<AZ::Transform, const char*> ROS2PoseControl::GetCurrentTransformViaTF2(
```

Comment:
I am not big fan of const char* in this context. 
Why? I do not know who is owner of this pointer and lifetime. 
As API user I do know if it is safe to use, when it will be freed, if should I free it by myself. Please adjust.